tldw_Server_API/app/api/v1/endpoints/media/process_ebooks.py

```
@@ -0,0 +1,424 @@
+        # Ensure overrides and derived fields are present even if the library
+        # omitted them (legacy parity expectations in tests).
+        result_dict.setdefault("metadata", {})
+        if title_override:
+            result_dict["metadata"]["title"] = title_override
+        if author_override:
+            result_dict["metadata"]["author"] = author_override
+        result_dict["keywords"] = keywords or result_dict.get("keywords") or []
+        result_dict.setdefault("analysis", None)
+        result_dict.setdefault("analysis_details", {})
+        return result_dict
+    except Exception as exc:  # pragma: no cover - defensive fallback
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)